src/tools/gophish_test.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-"""GoPhish Testing Tool for a Phishing Campaign Assessment (PCA).
+"""Send a mirror assessment to custom targets.
```

Comment:
I don't know what "a mirror assessment" is- can that language be clarified so that someone less familiar with the PCA process will understand it?